content/en/certificates.md

```diff
@@ -120,7 +120,7 @@ These intermediate CAs have currently-valid certificates, but are not being issu
 <details>
 <summary>Retired</summary>
 
-These intermediate CAs are no longer being used to issue Subscriber certificates. Those which still have valid certificates may be producing OCSP responses and/or CRLs.
+These intermediate CAs are no longer being used to issue Subscriber certificates. Those which still have valid certificates may be producing CRLs.
```

Comment:
A more technically correct sentence here would be: those that haven't yet expired and issued certificates in the past are still producing CRLs. Not a big deal, though.